src/store/index.js

```diff
@@ -25,6 +31,12 @@ export default createStore({
         },
         [LOGOUT](state) {
             state.isLoggedIn = false;
+        },
+        [CONFIG](state) {
+            state.pending = true;
+        },
+        [CONFIG_SUCCESS](state) {
+            state.pending = false;
```

Comment:
Hmm.. makes sense makes sense.

Change variable to `configFetchedPending` (or something similar) so it's clear what is pending